design/media_types.go

```diff
@@ -199,6 +199,21 @@ var User = a.MediaType("application/vnd.user+json", func() {
 	})
 })
 
+// Identity represents an identified user object
+var Identity = a.MediaType("application/vnd.identity+json", func() {
+	a.TypeName("Identity")
+	a.Description("ALM User Identity")
+	a.Attribute("id", d.String, "unique id for the user identity")
+	a.Attribute("fullName", d.String, "The users full name")
+	a.Attribute("imageURL", d.String, "The avatar image for the user")
+
+	a.View("default", func() {
+		a.Attribute("id")
+		a.Attribute("fullName")
+		a.Attribute("imageURL")
+	})
```

Comment:
You could convert this to json api as being done here:
https://github.com/almighty/almighty-core/pull/478/files#diff-7d67dc444801d745cdc248547dcce7c7R203